plan/physical_plans.go

```diff
@@ -117,8 +145,14 @@ func (p *physicalTableSource) addTopN(prop *requiredProperty) bool {
 	}
 	count := int64(prop.limit.Count + prop.limit.Offset)
 	p.LimitCount = &count
-	for _, item := range prop.props {
-		p.SortItems = append(p.SortItems, sortByItemToPB(p.client, item.col, item.desc))
+	for _, prop := range prop.props {
+		item := sortByItemToPB(p.client, prop.col, prop.desc)
+		if item == nil {
+			p.clearForTopnPushDown()
+			return false
```

Comment:
I didn't catch your point. You mean define the return value of clear***? It makes no sense.